chunk-webpack-plugin.js

```diff
@@ -39,7 +39,7 @@ function ChunkWebpackPlugin(options) {
 		throw new Error('The name of the target ("to") chunk cannot also be in the "from" chunk name list.');
 	}
 
-	base.call(this, this.targetChunkName, this.targetChunkName + '.js');
+	base.call(this, { name: this.targetChunkName, filename: this.targetChunkName + '.js' });
```

Comment:
We don't need to override `filename` template here cause it would be taken from  `webpack.config.js`: `output.filename: string`<br/>
For example:
```js
output: {
    filename: '[name].[hash:8].js'
}
```